### PR TITLE
Jan SLF presentation skeleton

### PR DESCRIPTION
**Pull request by remyb2canada**
First stab at it.
issue #321 
https://sara-sabr.github.io/util-presentation/presentation.html?url=https://raw.githubusercontent.com/sara-sabr/ITStrategy/slfpres/presentations/en/SLF.md

### CODE REVIEW

File: presentations/en/SLF.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+## 1. IT Strategy approach
+
+Purpose: Set the stage by clearly communicate that we are not doing conventional IT strategy with some examples of what that means
+
+Talk about:
+
+- Genesis of IT strategy and reason why the team got created (Dec 2018 senior leader meeting, goal of getting an integrated business-IT strategy)
+- How we are **NOT** doing this
+- We instead will focus on IT's ability to quickly respond to business changes
+- Talk about premise: the relationship between business and IT MUST change to a shared accountability one as the conventional contracting model (requirements based, a company within a company) of business-IT does not work and must move to a partnership one (outcomes based, one team). The future is too unpredictable and complex so the conventional long term planning do not work, we must instead reward an empirical cycle of trying, observing, and correcting.
```
Comment (gcharest) [inline presentations/en/SLF.md]: I'd probably define what "do not work" means to avoid having anyone take this as "we're not doing our jobs".

I don't think that's what we mean here but rather that in the Government, including ESDC, layers upon layers of governance with the silo'd approach has been used to try to manage risk but it has lead to the opposite effect.

File: presentations/en/SLF.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+Quote of Mark Schwartz
+> To work effectively in the digital world, you must first accept complexity and uncertainty, for they demand very different approach to carrying out initiatives. A predictable world rewards advanced planning and rigid plan execution. But a complex and uncertain world rewards an empirical cycle of trying, observing, and correcting.
+
+Talk about 4 angles the team is addressing using strategies (mandate)
+
+- Diagnose and Recommend changes on problems that inhibit the desired changes in behaviours
+  - example: Heavy Governance promotes risks, need to reduce burden of giving permission to work
```
Comment (gcharest) [inline presentations/en/SLF.md]: Flip the control with guardrails, auditable solutions (Git offers full view of changes and by whom), automated compliance (CI tests), etc.

**Comment by smellems** [comment]
Some slides have to much text.  I see this is mostly sample text explaining what should be in the slides.  I added some seperators, but some slides still have too much text and should be split

**Comment by remyb2canada** [comment]
> 
> 
> Some slides have to much text. I see this is mostly sample text explaining what should be in the slides. I added some seperators, but some slides still have too much text and should be split

yes, text is only sample. To start a conversation between ourselves as to what content we should add. Not the actual stuff that will be presented

File: presentations/en/SLF.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+### 3. Recommended immediate actions
+
+1. Continuous Learning (20% dedicated to learn, automate, and improve)
+2. Make ongoing work and backlog visible, including code
+3. Allow onboarding of modern tools, especially open source ones
+4. Avoid toil, invest in automation
+5. Promote smaller investments over large ones by leveraging Lite Projects
+
+Link to [Strategies and Actions](https://sara-sabr.github.io/ITStrategy/strategies-actions.html)
```
Comment (smellems) [inline presentations/en/SLF.md]: We should introduce our Quarterly strategies (actions) strategy.  Present the first one that will start Q1, 20% time continuous improvement. Talk about the [Goals](https://github.com/sara-sabr/ITStrategy/blob/master/_pages/en/strategy-learning-automating-improving.md#goals) and [IITB teams must (actions)](https://github.com/sara-sabr/ITStrategy/blob/master/_pages/en/strategy-learning-automating-improving.md#iitb-teams-must).

Comment (jaysonmc) [reply presentations/en/SLF.md]: I added a blurb about this here 

https://github.com/sara-sabr/ITStrategy/pull/515/commits/62beee1d5a3a5dfdef935bb4369d17b1430d44df

**Comment by gcharest** [comment]
> > Some slides have to much text. I see this is mostly sample text explaining what should be in the slides. I added some seperators, but some slides still have too much text and should be split
> 
> yes, text is only sample. To start a conversation between ourselves as to what content we should add. Not the actual stuff that will be presented

We probably need to add those as notes then so that it makes the content much lighter.

File: presentations/en/SLF.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+  - example: Heavy Governance promotes risks, need to reduce burden of giving permission to work
+  - example: Senior Advisors release process mapping involving 13 different teams and up to 49 email exchanges, need for embedded teams
+- Measuring strategy progress using non-invasive means and making course correction from emperical evidence
+- Challenge current thinking and approaches
+  - example: No more status update meetings due to modern tools (KANBAN boards in the open)
+  - example: This presentation not working in IE and refusal to make it work in IE because we're focusing on modern tech
+  - example: Technical work needing to be reviewed by non technical staff for approval
+  - example: Defaults over standards (if an expert in their respective area proposes a different approach than the departmental 'default' they are free to do so -- perhaps that will become the next default)
+
+---
```
Comment (myp192) [inline presentations/en/SLF.md]: it says there are 4 angles. I counted 3..?

**Comment by remyb2canada** [comment]
> 
> 
> In the "Focused area" section: I would add some touch on user experience.
> 
>     * We design for users with users, iterate and improve frequently
> 
>     * We focus on client-centric service design and delivery, an approach that focuses on addressing client or user expectations, needs, challenges, and feedback.
>       How do we know what user needs? What is his expectations?
>       -Talk to user
> 
>     * Get feedback, identify user needs
>       -Come up with ideas
> 
>     * Design ideas, iterate and improve it

@ElminaIusifova I would rephrase your suggestion with:

- design for users with users, iterate and improve frequently
  - This is because IT Strategy team is not the one building solutions. We set the IITB organization towards being more productive and are focusing on empowering employees and teams.
  - then add as notes the other points you list (that we can talk about without being them on the slide).

- add that IT Strategy team will start engaging with IITB teams soon to get their feedback, as users, on what needs to happen to get there
